packages/bootleg-common/bin/bootleg-setup-ganache

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+cleanup_ganache() {
+  # Kill the ganache instance that we started (if we started one and if it's still running).
+  if [ -n "$ganache_pid" ] && ps -p $ganache_pid > /dev/null; then
+    echo "Stopping Ganache"
+    kill -9 $ganache_pid
```

Comment:
hmm... pretty aggressive. Is `-9` necessary?

Reply:
Yes, yes it is.

Reply:
Aggressive from the perspective of the running process... 😜Shooting it in the head vs asking nicely for it to die. http://man7.org/linux/man-pages/man7/signal.7.html